contracts/test/token/starkgate/behavior/ERC20.ts

```diff
@@ -3,18 +3,38 @@ import { StarknetContract, Account } from 'hardhat/types/runtime'
 import { uint256 } from 'starknet'
 import { toBN } from 'starknet/utils/number'
 import { TIMEOUT } from '../../../constants'
+import { expectInvokeError } from '../../../../test/utils'
```

Comment:
Maybe We can write it back to `import { expectInvokeError } from '../../../utils'`